VaticanBAV/et/BAVet117.xml

```diff
@@ -102,22 +60,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                           </seg>
                         </handNote>
                       </handDesc>
-                      <additions>
-                        <list>
-                          <item xml:id="a1">
-                            <locus from="1r" to="4r"/>
-                            <desc>The scribe began to copy <title type="complete" ref="LIT1295Dersan"/>, leaving space to paint an accompanying image of the archangel Michael.</desc>
-                          </item>
-                          <item xml:id="a2">
```

Comment:
since pen trials most probably appeared after the two parts were already circulating as one unit (later layer) such additions should stay with the upper (main) layer, and not be split to the msparts simply by locus